terminal.go

```diff
@@ -238,8 +246,12 @@ func (g *LightningTerminal) Run() error {
 	// server is started.
 	g.rpcProxy = newRpcProxy(
 		g.cfg, g, g.validateSuperMacaroon, g.permsMgr, g.subServerMgr,
+		g.statusMgr,
 	)
 
+	litrpc.RegisterProxyServer(g.rpcProxy.grpcServer, g.rpcProxy)
+	litrpc.RegisterStatusServer(g.rpcProxy.grpcServer, g.statusMgr)
```

Comment:
nit: could use a comment for this code block, given that the all the other code blocks in this function are preceded by comments.